packages/preset-base/src/conditions.ts

```diff
@@ -57,8 +57,8 @@ export const conditions = {
 
   indeterminate: '&:is(:indeterminate, [data-indeterminate], [aria-checked=mixed], [data-state="indeterminate"])',
   required: '&:is(:required, [data-required], [aria-required=true])',
-  valid: '&:is(:valid, [data-valid])',
-  invalid: '&:is(:invalid, [data-invalid])',
+  valid: '&:is(:valid, [data-valid], [aria-invalid="false"])',
+  invalid: '&:is(:invalid, [data-invalid], [aria-invalid="true"])',
```

Comment:
Let's add only these. The placement conditions can be added in your userland code